go/pkg/diskcache/diskcache.go

```diff
@@ -0,0 +1,467 @@
+// Package diskcache implements a local disk LRU CAS cache.
+package diskcache
+
+import (
+	"container/heap"
+	"context"
+	"fmt"
+	"io"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"sync"
+	"sync/atomic"
+	"time"
+
+	"github.com/bazelbuild/remote-apis-sdks/go/pkg/digest"
+	"golang.org/x/sync/errgroup"
+	"google.golang.org/protobuf/proto"
+
+	repb "github.com/bazelbuild/remote-apis/build/bazel/remote/execution/v2"
+	log "github.com/golang/glog"
+)
+
+type key struct {
+	digest digest.Digest
+	isCas  bool
```

Comment:
Is isCas meant to do a performance optimization later? I don't see any behavior change based on this member.